src/Sentry.Unity.Android/SentryJava.cs

```diff
@@ -40,6 +43,88 @@ internal class SentryJava : ISentryJava
 {
     private static AndroidJavaObject GetSentryJava() => new AndroidJavaClass("io.sentry.Sentry");
 
+    public bool IsEnabled(IJniExecutor jniExecutor)
+    {
+        return jniExecutor.Run(() =>
+        {
+            using var sentry = GetSentryJava();
+            return sentry.CallStatic<bool>("isEnabled");
+        });
+    }
+
+    public bool Init(IJniExecutor jniExecutor, SentryUnityOptions options)
+    {
+        jniExecutor.Run(() =>
+        {
+            using var sentry = new AndroidJavaClass("io.sentry.android.core.SentryAndroid");
+            using var context = new AndroidJavaClass("com.unity3d.player.UnityPlayer")
+                .GetStatic<AndroidJavaObject>("currentActivity");
+
+            sentry.CallStatic("init", context, new AndroidOptionsConfiguration(androidOptions =>
+            {
+                androidOptions.Call("setDsn", options.Dsn);
+                androidOptions.Call("setDebug", options.Debug);
+                androidOptions.Call("setRelease", options.Release);
+                androidOptions.Call("setEnvironment", options.Environment);
+
+                var sentryLevelClass = new AndroidJavaClass("io.sentry.SentryLevel");
+                var levelString = GetLevelString(options.DiagnosticLevel);
+                var sentryLevel = sentryLevelClass.GetStatic<AndroidJavaObject>(levelString);
+                androidOptions.Call("setDiagnosticLevel", sentryLevel);
+
+                if (options.SampleRate.HasValue)
+                {
+                    androidOptions.SetIfNotNull("setSampleRate", options.SampleRate.Value);
+                }
+
+                androidOptions.Call("setMaxBreadcrumbs", options.MaxBreadcrumbs);
+                androidOptions.Call("setMaxCacheItems", options.MaxCacheItems);
+                androidOptions.Call("setSendDefaultPii", options.SendDefaultPii);
+                // Note: doesn't work - produces a blank (white) screenshot
+                androidOptions.Call("setAttachScreenshot", false);
+                androidOptions.Call("setEnableNdk", options.NdkIntegrationEnabled);
+                androidOptions.Call("setEnableScopeSync", options.NdkScopeSyncEnabled);
+                androidOptions.Call("setEnableAutoSessionTracking", false);
+                androidOptions.Call("setEnableActivityLifecycleBreadcrumbs", false);
+                androidOptions.Call("setAnrEnabled", false);
+                androidOptions.Call("setEnableScopePersistence", false);
```

Comment:
Consider spliting it more clearly like this:

```suggestion
                androidOptions.Call("setEnableNdk", options.NdkIntegrationEnabled);
                androidOptions.Call("setEnableScopeSync", options.NdkScopeSyncEnabled);

                // Options that are not to be set by the user

                // Note: doesn't work - produces a blank (white) screenshot
                androidOptions.Call("setAttachScreenshot", false);
                androidOptions.Call("setEnableAutoSessionTracking", false);
                androidOptions.Call("setEnableActivityLifecycleBreadcrumbs", false);
                androidOptions.Call("setAnrEnabled", false);
                androidOptions.Call("setEnableScopePersistence", false);
```